cmd/createcluster.go

```diff
@@ -393,7 +393,7 @@ func validateCreateConfig(ctx context.Context, conf clusterConfig) error {
 		return errors.New("number of operators is below minimum", z.Int("operators", conf.NumNodes), z.Int("min", minNodes))
 	}
 
-	if len(conf.ConsensusProtocol) > 0 && !consensus.IsSupportedProtocolName(conf.ConsensusProtocol) {
+	if len(conf.ConsensusProtocol) > 0 && !protocols.IsSupportedProtocolName(conf.ConsensusProtocol) {
```

Comment:
Do we error out if no consensus protocol has been chosen? Should we? Or it's better to leave a sensible default?

Reply:
The default, or "the last resort" is always QBFT v2.0. We error out here only if user specified something unknown, something we cannot find among the protocols supported by present implementation.

Reply:
Noted!